ironic/tests/unit/drivers/modules/drac/test_raid.py

```diff
@@ -926,19 +1112,37 @@ def test_create_configuration_with_backing_physical_disks(
     @mock.patch.object(drac_common, 'get_drac_client', spec_set=True,
                        autospec=True)
     @mock.patch.object(drac_raid, 'list_physical_disks', autospec=True)
+    @mock.patch.object(drac_raid, 'change_physical_disk_state', spec_set=True,
+                       autospec=True)
+    @mock.patch.object(drac_job, 'list_unfinished_jobs', spec_set=True,
+                       autospec=True)
+    @mock.patch.object(drac_job, 'wait_for_jobs_completion', spec_set=True,
+                       autospec=True)
     @mock.patch.object(drac_job, 'validate_job_queue', spec_set=True,
                        autospec=True)
     @mock.patch.object(drac_raid, 'commit_config', spec_set=True,
                        autospec=True)
     def test_create_configuration_with_predefined_number_of_phyisical_disks(
-            self, mock_commit_config, mock_validate_job_queue,
+            self,  mock_change_physical_disk_state,
```

Comment:
The order of the parameters in the method signature needs to be in reverse order of the decorators above, so the order of the parameters in the method signature should be:
            self, mock_commit_config, mock_validate_job_queue,
            mock_wait_for_jobs_completion, mock_list_unfinished_jobs,
            mock_list_physical_disks, mock_get_drac_client